programs/cryptid/src/instructions/close_transaction.rs

```diff
@@ -0,0 +1,122 @@
+use crate::error::CryptidError;
+use crate::instructions::util::*;
+use crate::state::did_reference::DIDReference;
+use crate::state::transaction_account::TransactionAccount;
+use crate::state::transaction_state::TransactionState;
+use crate::util::*;
+use anchor_lang::prelude::*;
+
+#[derive(Accounts)]
+#[instruction(
+/// A vector of controller account indices and their associated DID authority keys (to allow for generative cases).
+controller_chain: Vec<DIDReference>,
+/// The bump seed for the Cryptid signer
+cryptid_account_bump: u8,
+/// Index of the cryptid account
+cryptid_account_index: u32,
+/// The bump seed for the Did Account
+did_account_bump: u8,
+)]
+pub struct CloseTransaction<'info> {
+    /// The Cryptid instance to execute with
+    /// CHECK: Cryptid Account can be generative and non-generative
+    #[account(
+    mut,
+    // TODO(ticket): Verification done in instruction body. Move back with Anchor generator
```

Comment:
should this  be done?

Reply:
There are 10x of the exact TODOs. It's a TODO that get's resolved when anchor provides the functionality.